db/data/regions/GP.yml

```diff
@@ -1,6 +1,8 @@
 ---
 name: Guadeloupe
 code: GP
+code_alternates:
+- FR-GP
```

Comment:
I think some of the alternate codes I added for france may be a bit more debated, or at least they're implemented in Worldwide differently.